### PR TITLE
fix missing repository enablement segments for katello

### LINKED ISSUE #422: Katello Nightly / 4.0 docs missing yum commands for foreman-release and katello-repos
If you navigate to the install docs to try and install 4.0 here:

https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-katello.html#configuring-repositories_foreman

Under **Section 2.1** you will note that there are no actual 'yum install <foreman-release.rpm>' or 'yum install <katello-repos>' commands that are required to configure before you can install Foreman and Katello.

Also note the formatting errors:

```

+ . Clear any metadata:
+

```

### PR DESCRIPTION
**Pull request by jlsherrill**
Cherry-pick into:

* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

Closes #422

### CODE REVIEW

**Comment by jlsherrill** [comment]
closes #422

**Review by melcorr — CHANGES_REQUESTED** [review]
One small thing and I'll merge @jlsherrill

File: guides/common/modules/proc_configuring-repositories.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Use this procedure to enable the repositories that are required to install {ProductName}.
 
-.Procedure
+.Red Hat Enterprise Linux Users
```
[the remark this answers is not in the document]
Comment (jlsherrill) [reply guides/common/modules/proc_configuring-repositories.adoc:7]: So to me its confusing without it.  This configuration section is divided into two parts, Centos users and RHEL users.  Today the 'rhel' part is just labeled 'procedure' which i imagine centos users would try to follow. 

Thoughts?

But now i realize the rhel instructions are missing what repos to enable when installing katello.  let me fix that.

[the remark this answers is not in the document]
Comment (melcorr) [reply guides/common/modules/proc_configuring-repositories.adoc:7]: If you leave .Procedure, add a space and put 

`.For {RHEL} users`

That would be good for now. I'll think about this a bit more.

[the remark this answers is not in the document]
Comment (jlsherrill) [reply guides/common/modules/proc_configuring-repositories.adoc]: is this what you meant @melcorr ?

**Review by melcorr — APPROVED** [review]

File: guides/common/modules/proc_configuring-repositories.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (melcorr) [reply guides/common/modules/proc_configuring-repositories.adoc]: It is, thanks!

**Pull request merged**